jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1098,6 +1098,30 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _sort(*operand: tf.Tensor, dimension: int = -1,
+          is_stable: bool = True) -> Tuple[TfVal, ...]:
+  if len(operand) == 0:
```

Comment:
This is probably more like an internal assertions, because we should never have a `sort_p` primitive with no arguments (if we do, the `TypeError` should be in `lax.sort`)

Reply:
However, if you do want to keep it also for extra safety, I think it is fine.